### PR TITLE
chore(CSS) font-tech() and font-format() functions added in Fx 106

### PR DESCRIPTION
**Pull request by bsmth**
This PR adds some compat data under `@supports` CSS for:

* `font-tech()` does a user agent support a specific [font technology](https://www.w3.org/TR/css-fonts-4/#font-tech-definitions)?
* `font-format()` does the user agent support certain [font formats](https://www.w3.org/TR/css-fonts-4/#font-format-definitions)?

Support at the moment seems to only be functional in Fx 106.

### Related issues

- [ ] Parent issue: https://github.com/mdn/content/issues/20917

### CODE REVIEW

File: css/at-rules/supports.json [hunk 1]
```diff
@@ -79,6 +79,76 @@
               "deprecated": false
             }
           }
+        },
+        "font-technology": {
```
Comment (teoli2003) [inline css/at-rules/supports.json:83]:
```suggestion
        "font-tech": {
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: css/at-rules/supports.json (hunk 1 above)
Comment (bsmth) [reply css/at-rules/supports.json:83]: Thanks 👍🏻

**Review by teoli2003 — APPROVED** [review]
👍

File: css/at-rules/supports.json [hunk 2]
```diff
@@ -44,6 +44,76 @@
...
         },
+        "font-format": {
+          "__compat": {
+            "description": "<code>font-format()</code>",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": "mirror",
+              "edge": false,
```
Comment (teoli2003) [inline css/at-rules/supports.json:55]:
```suggestion
              "edge": {
                "version_added": false
              },
```

Same for the others.

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: css/at-rules/supports.json (hunk 2 above)
Comment (bsmth) [reply css/at-rules/supports.json]: ah-ha, thanks for the hint

**Comment by bsmth** [comment]
Going to merge after fighting with CI 🤖
Thanks @teoli2003

**Pull request merged**